core/traffic_class.cc

```diff
@@ -53,6 +53,27 @@ bool PriorityTrafficClass::AddChild(TrafficClass *child, priority_t priority) {
   return true;
 }
 
+bool PriorityTrafficClass::RemoveChild(TrafficClass *child) {
+  if (child->parent_ != this) {
+    return false;
+  }
+
+  for (auto it = children_.begin(); it != children_.end(); it++) {
```

Comment:
Sure, I've changed the code here, and in two other places that had the same pattern.